Dockerfile

```diff
@@ -26,8 +26,8 @@ RUN \
   echo "**** install budge ****" && \
   mkdir -p /app/budge && \
   if [ -z ${BUDGE_RELEASE+x} ]; then \
-    BUDGE_RELEASE=$(curl -sX GET "https://api.github.com/repos/linuxserver/BudgE/commits/main" \
-    | awk '/sha/{print $4;exit}' FS='[""]'); \
+    BUDGE_RELEASE=$(curl -sX GET "https://api.github.com/repos/linuxserver/BudgE/releases" \
```

Comment:
I believe this should be `releases/latest` otherwise it will return all release tags (currently there is only one so it works for now)

Reply:
Ah, good point. Pushed an update.